backend/elasticsearch/src/main/java/com/rbmhtechnology/vind/elasticsearch/backend/util/ElasticMappingUtils.java

```diff
@@ -0,0 +1,114 @@
+package com.rbmhtechnology.vind.elasticsearch.backend.util;
+
+import com.rbmhtechnology.vind.SearchServerException;
+import com.rbmhtechnology.vind.elasticsearch.backend.ElasticSearchServer;
+import com.rbmhtechnology.vind.elasticsearch.backend.MappingValidationException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class ElasticMappingUtils {
+
+    private static final Logger log = LoggerFactory.getLogger(ElasticMappingUtils.class);
+
+    public static String getDefaultMapping() {
+        final Path mappingsFile = Paths.get(ElasticMappingUtils.class
+                .getClassLoader().getResource("mappings.json").getPath());
```

Comment:
should be a static final constant (I know it is handled from the compiler like this anyways, but is more obvious